exporter/README.md

```diff
@@ -85,7 +85,18 @@ queued-exporters:
     exporters:
       opencensus:
         endpoint: "127.0.0.1:55566"
+        cert-pem-file: "server_ca_public.pem" # PEM file used to enable TLS. For trusted CAs from
+                                              # system pool use "secure:" setting, see below.
         compression: "gzip"
+        reconnection-dealy: 2s # Delay (+70% jitter) before reconnection attempt in case of error.
+        secure: true # Used to export to destinations trusted by certificates from the system pool.
+                     # "cert-pem-file:" takes precedence over this setting.
+        keepalive:
+          # Keepalive settings for gRPC clients, see https://godoc.org/google.golang.org/grpc/keepalive#ClientParameters.
+          # Recommended to be set for cases that need to support bursts of data and periods of inactivity.
+          time: 30s # After inactive for this amount to time client will send a ping to the server.
```

Comment:
Suggestion: consider `interval` or `period`.

Reply:
I also don't like `time` very much (`interval` is better to me) but I'm following the names at https://godoc.org/google.golang.org/grpc/keepalive#ClientParameters so users can read that doc and directly map to the configuration items, if we start to diverge one needs to mentally map them.

Reply:
Make sense to me.